src/views/account/UserProfile.vue

```diff
@@ -75,6 +129,31 @@ const updateProfile = (e: MouseEvent) => {
   });
 };
 
+const updatePassword = (e: MouseEvent) => {
+  e.preventDefault();
+
+  passwordFormRef.value?.validate(async (errors) => {
+    if (!errors) {
+      loading.value = true;
+
+      const response = await fetch(`${baseURL}/auth/password/change`, {
+        body: JSON.stringify(passwordForm.value),
+        method: "POST",
+      });
+
+      loading.value = false;
+      if (!response.ok) {
+        push.error("Something went wrong, please try again later");
+        throw new Error("Password error");
+      }
```

Comment:
logout on a successful password change. You can just navigate to /auth/logout for this

Reply:
Maybe add a notification/message and then delay the navigation to logout for about 1.5 seconds